src/pushNotifications/TokenHandling.ts

```diff
@@ -0,0 +1,75 @@
+import { Platform } from 'react-native';
+
+import { secrets } from '../config';
+import * as SecureStore from 'expo-secure-store';
+import * as appJson from '../../app.json';
+
+
```

Comment:
please avoid unnecessary line breaks. eslint should automatically fix this.

Reply:
done in 5c4968a26c167de3f0343c155960812f53460942

eslint did not catch this

Reply:
sad. i will try things out regarding this.